content-review/impl/turnitin-oc/src/main/java/org/sakaiproject/contentreview/turnitin/oc/ContentReviewServiceTurnitinOC.java

```diff
@@ -932,6 +939,19 @@ public boolean incrementItem(ContentReviewItem item) {
 		}
 		return true;
 	}
+	
+	private boolean checkForDraft(ContentReviewItem item, Assignment assignment) {
```

Comment:
I think the name of this function needs to be changed or the return value needs to be changed. I expect the "checkForDraft" to return true if it is a draft and false if it is not. This seems to be the opposite.